alderaan/pipeline-scripts/prometheus.groovy

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+def pipeline_id = env.BUILD_ID
+def node_label = NODE_LABEL.toString()
+def prometheus_test = PROMETHEUS_TEST.toString().toUpperCase()
+def property_file_name = "prometheus.properties"
+
+println "Current pipeline job build id is '${pipeline_id}'"
+
+// run prometheus scale test
+stage ('prometheus_scale_test') {
+	if ( prometheus_test == "TRUE") {
+		currentBuild.result = "SUCCESS"
+		node(node_label) {
+			if (fileExists(property_file_name)) {
+				println "prometheus.properties file exist... deleting it..."
+				sh "rm ${property_file_name}"
+			}
+			sh "wget -O ${property_file_name} ${PROMETHEUS_PROPERTY_FILE}"
+			sh "cat ${property_file_name}"
+			def prometheus_properties = readProperties file: property_file_name
+			def REFRESH_INTERVAL = prometheus_properties['REFRESH_INTERVAL']
```

Comment:
All the parameters need by the job should be read from the properties including jump_host, user, containerized e.t.c